### DISCUSSION TITLE
Where did the source of this image come from?

### DISCUSSION THREAD

**Discussion by blackinitial** [mearashadowfax/ScrewFast#127] [General]
I see that some images use image platforms that have been optimized and transformed from unsplash. I think it's easier to optimize the image load in the future on my website.
like a collection of avatar images:
```js
const avatarSrcs: Array<string> = [
  "https://images.unsplash.com/photo-1568602471122-7832951cc4c5?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=facearea&facepad=2&w=300&h=300&q=80",
  "https://images.unsplash.com/photo-1531927557220-a9e23c1e4794?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=facearea&facepad=2&w=300&h=300&q=80",
  "https://images.unsplash.com/photo-1541101767792-f9b2b1c4f127?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&&auto=format&fit=facearea&facepad=3&w=300&h=300&q=80",
  "https://images.unsplash.com/photo-1492562080023-ab3db95bfbce?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=facearea&facepad=2&w=300&h=300&q=80",
];
```
I want to use that technique on my website, where can I read the documentation for using it? is there a link?

**Comment by mearashadowfax** [comment]
Hello @blackinitial! I assume you are referring to Unsplash's API and its parameters such as `fit=facearea`, etc. You can check all available options for the images from Unsplash here: [Unsplash Documentation](https://unsplash.com/documentation#dynamically-resizable-images). I believe Unsplash is using [Imgix](https://www.imgix.com/) for image optimization. You can read more about it in their blog post: [The Service Powering Unsplash's Beautiful Imagery](https://unsplash.com/blog/the-service-powering-unsplashs-beautiful-imagery/).

**Reply by blackinitial** [reply]
thanks for the answer 👍🏻
